docs/source/resources/commands/conda-build.rst

```diff
@@ -176,20 +179,35 @@ conda-build
                  Disable force upload to anaconda.org, preventing overwriting any
                  existing packages
 
+          <B>--zstd-compression-level</B> {1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22}
```

Comment:
Given https://github.com/conda/conda-build/commit/53aa0d88ad03a5a05eeeee7fe2d9d99bf5d072cc I'm wondering if we should limit it to 19. We know that levels above 19 are not suited for memory consumption reasons.

Reply:
The tool still accepts levels up to 22 though: https://github.com/conda/conda-build/blob/ef2026a09fd5d8572e506937effb4eb88d3a08b7/conda_build/cli/main_build.py#L164

Perhaps we could change the description to something like:

```
When building v2 packages, set the compression level used by
conda-package-handling. Defaults to 19. Note that using levels
above 19 is not advised due to high memory consumption.
```

Reply:
@jezdez I've made the change suggested above, hopefully that's OK?

Reply:
Good idea, thank you for your patience!